src/gui/TimeLineWidget.cpp

```diff
@@ -403,7 +404,7 @@ void TimeLineWidget::mouseMoveEvent( QMouseEvent* event )
 			}
 			else
 			{
-				m_loopPos[i] = t.quantize(1.0);
+				m_loopPos[i] = t.quantize(m_snapSize);
 			}
 			// Catch begin == end
 			if( m_loopPos[0] == m_loopPos[1] )
```

Comment:
I guess this should not be done when Ctrl is being pressed (see my other comment).